src/Environment/EnvironmentVariables.php

```diff
@@ -110,9 +113,9 @@ private static function getenv(string $key): string
      */
     private static function getenvWithFallback(string $key, string $default): string
     {
-        $value = getenv($key);
+        $value = Env\get_var($key);
 
-        return $value === false || $value === '' ? $default : $value;
+        return empty($value) ? $default : $value;
```

Comment:
`empty()` should really never be used, since `'0'` also matches

Reply:
oh, my bad :)